### PR TITLE
Deprecate AU Base Diagnostic Request, create AU Base Service Request (FHIR-46714)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46714](https://jira.hl7.org/browse/FHIR-46714)

* Deprecate AU Base Diagnostic Request
* Create AU Base Service Request
* Point relevant examples to new Service Request resource
* Updates to Terminology table
* Change log entry

### CODE REVIEW

File: input/resources/au-servicerequest.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <element id="ServiceRequest.code">
+      <path value="ServiceRequest.code" />
+      <binding>
+        <strength value="example" />
+        <valueSet value="https://healthterminologies.gov.au/fhir/ValueSet/procedure-1" />
+      </binding>
+    </element>
+    <element id="ServiceRequest.performerType">
+      <path value="ServiceRequest.performerType" />
+      <binding>
```
Comment (dbojicic) [inline input/resources/au-servicerequest.xml:60]: This is inherited from core ServiceRequest resource, no need to include in the profile.

File: input/resources/au-servicerequest.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <path value="ServiceRequest.identifier" />
+      <short value="Identifiers assigned to this order" />
+      <definition value="Identifiers assigned to this order instance by the orderer and/or the receiver and/or order fulfiller." />
+      <type>
+        <code value="Identifier" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/Identifier" />
+        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-localorderidentifier" />
+      </type>
+    </element>
+    <element id="ServiceRequest.code">
```
Comment (dbojicic) [inline input/resources/au-servicerequest.xml:51]: Missing constraint on ServiceRequest.requisition.

File: input/examples/servicerequest-hepatitis-b-antibody.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <ServiceRequest xmlns="http://hl7.org/fhir">
     <id value="hepatitis-b-antibody"/>
     <meta>
-        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-diagnosticrequest"/>
```
Comment (dbojicic) [inline input/examples/servicerequest-hepatitis-b-antibody.xml:4]: This change is not in scope of https://jira.hl7.org/browse/FHIR-46714.

File: input/examples/servicerequest-path-example0.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <ServiceRequest xmlns="http://hl7.org/fhir">
     <id value="path-example0"/>
     <meta>
-        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-diagnosticrequest"/>
+        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-servicerequest"/>
```
Comment (dbojicic) [inline input/examples/servicerequest-path-example0.xml:4]: This change is not in scope of https://jira.hl7.org/browse/FHIR-46714.

File: input/pagecontent/changes.md [hunk 5]
```diff
@@ -92,6 +92,7 @@ To help implementers, only the more significant changes are listed here.
       <li>Added invariant inv-pra-6 to RSG.sourceDocument.type to bind Jurisdiction ValueSet - AU Extended (extensible) (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>).</li>
     </ul>
   </li>
+  <li>Deprecated <a href="StructureDefinition-au-diagnosticrequest.html">AU Base Diagnostic Request</a> (<a href="https://jira.hl7.org/browse/FHIR-46080">FHIR-46080</a>).</li>
```
Comment (dbojicic) [inline input/pagecontent/changes.md:95]: Linked to incorrect JIRA, please change to point to FHIR-46714.

File: input/ig.xml [hunk 6]
```diff
@@ -890,15 +890,15 @@
 			</reference>
 			<name value="ServiceRequest - for Hepatitis B surface antibody measurement"/>
 			<description value="Service request for Hepatitis B surface antibody measurement"/>
-			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-diagnosticrequest"/>
+			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-servicerequest"/>
```
Comment (dbojicic) [inline input/ig.xml:893]: This change is not in scope of https://jira.hl7.org/browse/FHIR-46714.

File: input/ig.xml [hunk 7]
```diff
@@ -890,15 +890,15 @@
...
 		</resource>
 		<resource>
 			<reference>
 				<reference value="ServiceRequest/path-example0"/>
 			</reference>
 			<name value="ServiceRequest - with multiple result copies to indicated"/>
 			<description value="A service request with multiple result copies to indicated"/>
-			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-diagnosticrequest"/>
+			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-servicerequest"/>
```
Comment (dbojicic) [inline input/ig.xml:901]: This change is not in scope of https://jira.hl7.org/browse/FHIR-46714.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments

File: input/pagecontent/changes.md (hunk 5 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:95]: Amended, thanks.

File: input/resources/au-servicerequest.xml (hunk 1 above)
Comment (JayMurdoch) [reply input/resources/au-servicerequest.xml:60]: Removed, thanks.

File: input/resources/au-servicerequest.xml (hunk 2 above)
Comment (JayMurdoch) [reply input/resources/au-servicerequest.xml:51]: Added, thanks.

File: input/resources/au-servicerequest.xml (hunk 1 above)
Comment (dbojicic) [reply input/resources/au-servicerequest.xml]: Confirmed

File: input/resources/au-servicerequest.xml (hunk 2 above)
Comment (dbojicic) [reply input/resources/au-servicerequest.xml:59]: Confirmed

File: input/pagecontent/changes.md (hunk 5 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Confirmed

**Review by dbojicic — APPROVED** [review]

**Review by brettesler-ext — APPROVED** [review]

**Pull request merged**